docs/authentication/hosted-add-authentication.mdx

```diff
@@ -0,0 +1,313 @@
+
+## Overview
+
+With this guide you will:
+
+1. Create a Beyond Identity application using the Hosted Web configuration type.
+1. Configure an OpenID Connect client library or build your own.
+
+## Create your Beyond Identity application
+
+### Prerequisites
+
+- [x] A [developer account](https://www.beyondidentity.com/developers)
+- [x] Logged into your admin console
+
+Depending on the region you chose when you signed up, you can access the Beyond Identity admin console for your tenant at:   
+- [BI admin console US](https://console-us.beyondidentity.com/login)  
+- [BI admin console EU](https://console-eu.beyondidentity.com/login)
+
+:::tip  
+_If you are signing in from a different computer or browser, you can enter the email address you used when you signed up. Beyond Identity will send you a one time link you can use to sign in and enroll a passkey from the new browser._  
+:::
+
+### Create a new realm
+
+So far, you have the Beyond Identity Admin Realm for your Beyond Identity tenant only. In this step, you'll create a new Realm to hold your users' identities and the configuration. 
+
+import CreateRealmConsole from '../includes/_create-realm-console.mdx';
+
+<CreateRealmConsole />
+
+### Create an application
+
+In this step, you'll create a new Application containing the configuration for your users.
+
+import AddAppAdminConsole from '../includes/_add-application-console.mdx';
+
+<AddAppAdminConsole />
+
+On the **External Protocol** tab, use the following values to complete this tab.
+
+  | Property | Value |
+  | --- | --- |
+  | **Display Name** | Descriptive name you choose |
+  | **Protocol** | OIDC<br /><ProtocolOIDC/> |
+  | **Client Type** | Confidential<br /><ClientTypeConfidential/> |
+  | **PKCE** | S256<br /><PKCES256/> |
+  | **Redirect URIs** | Use a route in your app this is capable of handling the authorization code exchange.<br /><br />If you are using an OpenID Connect client library, a redirect URI may be specified for you. <br /><br />For example: In a Next.js application using NextAuth, your redirect URI follows the pattern:<br /><br /><pre>http://localhost:3000/api/auth/callback/beyondidentity</pre><div class= "note-message note-message-table"><p>where `beyondidentity` in this URI is the ID of the Auth provider as configured in the providers array in NextAuth.js and `/api/auth/callback/` is based on the Next.js route file structure.</p></div> |
+  | **Token Endpoint Auth Method** | Client Secret Basic<br /><TokenEndpointAuthMethodClientSecretBasic/> |
+  | **Grant Type** | Authorization Code<br /><GrantTypeAuthorizationCode/> |
+  | **All other options** | Use the default values for the remaining options |
+
+On the **Authenticator Config** tab, use the following values to complete this tab.
+
+   | Property | Value | 
+   | ----------- | ----------- |
+   | **Configuration Type** | Hosted Web <br /><HostedWeb/> |
+   | **Authentication Profile** | Use the default values for the remaining options |
```

Comment:
I've just noticed that the console UI options for Authentication profile look different when I'm adding a brand new app vs when I am changing an existing app from Embedded SDK back to Hosted Web. In the latter case there is no default value selected and there are no sub options under Hardware bound passkey. Is this as expected?

Reply:
I reworded this to "recommended" values in case the default doesn't show